includes/taxo.php

```diff
@@ -443,7 +443,7 @@ function is_question_tag() {
  * @return bool
  */
 function is_question_tags() {
-	if ( ap_get_tags_slug() === get_query_var( 'ap_page' ) ) {
+	if ( 'tags' === ap_current_page() ) {
```

Comment:
@mebishalnapit Any reason for removing `ap_get_tags_slug()` ?

Reply:
@rahularyan I think it was left as testing purpose since `get_query_var( 'ap_page' )` was not working I modified that to `ap_current_page()` and seems it is left as it is currently. Should I modify that function to the older one?